submit_compress.csh

```diff
@@ -45,7 +47,7 @@ source data_scripts.csh
 
 set comp_cmd      = 'gzip '
 # set comp_cmd      = 'gzip -k'
-set ymds          = 2014-12-01-00000
+set ymds          = 2016-02-01-00000
```

Comment:
Why was this 2014-12 and is now being changed to 2016-02? In other words, is this something that always needs to be updated and was neglected, or is it a special case here?